contracts/Orders.sol

```diff
@@ -1,56 +1,31 @@
 pragma solidity 0.4.18;
 
-
 import "./Utils2.sol";
```

Comment:
line space above

2 lines required

Reply:
@amitkot 
not sure.
I am sure its needed before contract.
but in all other contract its also before the imports. so consider doing it anyway...